### PR TITLE
Prepare beta release for Key Vault SDK

### PR DESCRIPTION
**Pull request by heaths**

### CODE REVIEW

**Review by jsquire — APPROVED** [review]

File: sdk/keyvault/Azure.Security.KeyVault.Certificates/CHANGELOG.md [hunk 1]
```diff
@@ -1,15 +1,11 @@
...
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 4.3.0-beta.3 (2022-01-11)
 
 ### Other Changes
 
+- Use operation helpers for long-running operations.
```
Comment (christothes) [inline sdk/keyvault/Azure.Security.KeyVault.Certificates/CHANGELOG.md:7]: Is this an internal implementation detail, or exposed in the API?

Comment (heaths) [reply sdk/keyvault/Azure.Security.KeyVault.Certificates/CHANGELOG.md:7]: Internal, but I had to put something in the changelog or the release scripts would err. I suppose I could just put something vague like "Updates to improve long-term support". @AlexGhiondea?

Comment (AlexGhiondea) [reply sdk/keyvault/Azure.Security.KeyVault.Certificates/CHANGELOG.md:7]: I wouldn't put this as it doesn't impact the customers.

Comment (heaths) [reply sdk/keyvault/Azure.Security.KeyVault.Certificates/CHANGELOG.md:7]: I need to include something in the changelog or the scripts complain.

Comment (heaths) [reply sdk/keyvault/Azure.Security.KeyVault.Certificates/CHANGELOG.md:7]: Per offline discussion, I'll submit another PR to change this to just "Bug fixes".

**Pull request merged**